assets/alertmanager/network-policy.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
I don't think we want to include NetworkPolicy (at least for now)

Reply:
@simonpasquier Excluded network policies, PTAL.